src/components/commons/Footer.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+<footer class="footer footer-center p-10 text-primary-content ossph-gradient-bg">
+  <div class="grid grid-flow-row sm:grid-flow-col gap-4">
+    <a href="https://discord.com/servers/open-source-software-ph-905496362982981723"
+      target="_blank">
+      <button class="btn btn-ghost text-white">
+        <i class="lab la-discord"></i>
+        Discord
+      </button>
+    </a>
+    <a href="https://www.facebook.com/ossph.org"
+      target="_blank">
+      <button class="btn btn-ghost text-white">
+        <i class="lab la-facebook"></i>
+        Facebook
+      </button>
+    </a>
+    <a href="https://twitter.com/OSSPhilippines"
+      target="_blank">
+      <button class="btn btn-ghost text-white">
+        <i class="lab la-twitter"></i>
+        Twitter
+      </button>
+    </a>
+    <a href="https://bit.ly/3xjDvN2"
+      target="_blank">
+      <button class="btn btn-ghost text-white">
+        <i class="la la-newspaper"></i>
+        Press Kit
+      </button>
+    </a>
+  </div>
+  <p>Copyright © 2023 Open Source Software PH. All Rights Reserved.</p>
```

Comment:
Make the year dynamic. You can use vue.js interpolation directly here. Like so:

```js
{{new Date().getFullYear()}}
```

Reply:
Sir @danielfpadilla in case you missed this one, just notifying you. Thank you <3

Reply:
Gosh I really missed this one lol

Reply:
This is done